src/base/system_parameter.c

```diff
@@ -2344,6 +2346,12 @@ static UINT64 prm_ddl_audit_log_size_lower = 10485760ULL;	/* 10M */
 static UINT64 prm_ddl_audit_log_size_upper = 2147483648ULL;	/* 2G */
 static unsigned int prm_ddl_audit_log_size_flag = 0;
 
+UINT64 PRM_COUNT_DEBUG = false;
+static UINT64 prm_count_debug_default = 0;	/* 10M */
+static UINT64 prm_count_debug_lower = 0;	/* 10M */
+static UINT64 prm_count_debug_upper = UINT64_MAX;	/* 2G */
```

Comment:
please ignore the comment :-)

Reply:
What does the 10M, 2G comment mean?

Reply:
There is no meaning. It is just copied from another code. This line will be erased before merging.